docs/ldml/tr35-general.md

```diff
@@ -1057,8 +1075,8 @@ grouping
     * The locale data for currency display names is supplied in the `currencies` element, not in the `units` element.
 
 Note that while the syntax allows for unit_constants in multiple places, the typical use case is only one instance, after a "-per-".
-The normalized form of a unit identifier has at most one unit_constant in the numerator and one in the denominator.
-For example, `2-kilowatt-7-hour-per-3-meter-5-second` has the equivalent normalized form `14-kilowatt-hour-per-15-meter-second`.
+The normalized, non-deprecated form of a unit identifier has at most one unit_constant in the denominator immediately after the per.
+For example, `kilowatt-hour-per-3-meter-5-second` has the equivalent normalized form `kilowatt-hour-per-15-meter-second`.
```

Comment:
Should the length restriction be mentioned here as well in the prose?

Reply:
It doesn't need to be. The syntax already contains a well-formed clause:

> * [ wfc: The string length of the unit constant must be less than 9 characters. ]